.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'New version type [major | minor | patch]'
+        required: true
+        default: 'patch'
+jobs:
+  bumpVersion:
+    runs-on: ubuntu-20.04
+    steps:
+      - run: |
+          echo "New version type: ${{ github.event.inputs.type }}"
+
+      - name: Branch Protection Bot - Temporarily disable "include administrators" branch protection
+        uses: benjefferies/branch-protection-bot@1.0.7
+        if: always()
+        with:
+          access_token: ${{ secrets.GH_RELEASE_TOKEN }}
+          enforce_admins: false
+          branch: master
+
+      - name: Setup checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GH_RELEASE_TOKEN }}
+
+      - name: Config git
+        run: |
+          git config --local user.email "62586190+process-analytics-bot@users.noreply.github.com"
+          git config --local user.name "process-analytics-bot"
+          git config pull.rebase true
+
+      - name: Checkout master
+        run: git checkout master && git pull --tag
+
+      - name: Read version from DESCRIPTION and set version variables
+        run: |
+          VER=$(grep Version DESCRIPTION | cut -d ': ' -f 2)
+          echo "CURRENT_VERSION=$VER" >> $GITHUB_ENV
+
+      - name: Generate new version
+        id: next_version
+        uses: leovs09/semver-release-action@v1
+        with:
+          bump: ${{ env.type }}
+          version: ${{ env.CURRENT_VERSION }}
+          github_token: ${{ secrets.GH_RELEASE_TOKEN }}
+          dry_run: true
+
+      - name: Update README.md
+        run: |-
+          sed -i -E 's/${{ env.CURRENT_VERSION }}/${{ steps.next_version.outputs.version }}/g' README.md
+
+      - name: Update DESCRIPTION
+        run: |-
+          sed -i -E 's/Version: "(.*)"/Version: "${{ steps.next_version.outputs.version }}"/g' DESCRIPTION
+
+      - name: Commit
+        run: |
+          git add README.md
+          git add DESCRIPTION
+          git commit -m "[RELEASE] Prepare version for release"
+          git push
+
+      - name: Tag ${{ steps.next_version.outputs.version }}
+        run: |
+          git tag -a v${{ steps.next_version.outputs.version }} -m "[RELEASE] ${{ steps.next_version.outputs.version }}"
```

Comment:
why did you choose to create an annotated git tag here?

Reply:
ℹ️ To add a message on the tag, it's the only way according the documentation : https://git-scm.com/book/en/v2/Git-Basics-Tagging.
According https://git-scm.com/docs/git-tag, if `-m <msg>` or `-F <file>` is given and `-a`, `-s`, and `-u <keyid>` are absent, `-a` is implied.

So I can remove it, but it is the same result 🙂

Reply:
fine